web/l10n/en.json

```diff
@@ -232,6 +232,8 @@
     "mapTitle": "Map",
     "mapLayer": "Map Layer",
     "mapCustom": "Custom Map",
+    "mapCustomXyzFormat": "Custom (XYZ)",
+    "mapCustomArcgisRest": "Custom (ArcGIS REST)",
```

Comment:
Can we drop "rest" to make this shorter?

Reply:
Of course. Good. Thanks.